### ISSUE TITLE
Volunteer vassalage is way too easy to demand

### ISSUE THREAD

**Issue by CppMaster** [LoneGazebo/Community-Patch-DLL#4851]
_1. Mod version (i.e Date - 4/23):_
10-10-2

_2. Mod list (if using Vox Populi only, leave blank):_

_3. Error description:_ 
In my current Carthage game at early Renaissance, I become master of every civ. Russia and Egypt capitulated to me, so no issue with that, but top 3 civs other than me agreed to become a vassal to me via demand. Khmer was Afraid, Inca was Neutral and Timur was Guarded for me. Vassalage via demand is way too easy to achieve. It may also be caused by how easy AI become Afraid without big military advantage.
@LoneGazebo  could you make AI to be less willing to become a vassal voluntarily? Or add an option in game settings to turn off volunteer vassalage.

_4. Steps to reproduce (optional):_

---------------------------
Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):
[screenshot](https://forums.civfanatics.com/attachments/easy-vassalage-png.506617/)

**Comment by matlajs** [comment]
yes please, lessen it.

**Comment by mikes61293** [comment]
In my Huge Diety Game as Russia, I was able to conquer about half of my continent by the early industrial era.  I moved my troops to Japan's borders and demanded vassalage which they agreed to, this made sense.  I was then able to demand vassalage from Greece and Denmark who were on my continent, but my army was nowhere near them, Greece I could have conquered but Denmark was on the other side of Greece and I had no navy.  I then demanded vassalage of The Huns, Siam, Assyria, The Iroquois all of whom were on on the other continent and definitely out of my reach. 

I will upload pictures when I get home.
